homework-submission/src/Components/list.css

```diff
@@ -0,0 +1,104 @@
+@keyframes rainbow {
+  0% {
+    color: rgb(209, 23, 23);
+  }
+  25% {
+    color: rgb(157, 17, 192);
+  }
+  50% {
+    color: rgb(11, 66, 155);
+  }
+  75% {
+    color: rgb(9, 110, 56);
+  }
+  100% {
+    color: rgb(179, 22, 17);
+  }
+}
+
+.no-todos {
+  font-size: 24px;
+  font-weight: 700;
+  min-width: 200px;
+  padding: 75px 0;
+  text-align: center;
+  margin: 0 auto;
+  color: #222;
+  animation: rainbow 10s infinite;
+  transform: scale(1.2);
+}
+
+.list {
+  display: flex;
+  flex-direction: column;
+  list-style-type: none;
+  width: 600px;
+  max-height: 70vh;
+  overflow-y: scroll;
+  margin: 50px auto;
+  padding: 50px 80px 70px 60px;

```

Comment:
⚠️ this is an antipattern. The padding should always be consistent and harmonic. You have different values for each side, this means the design is wrong or the implementation is wrong. 90% of the case the padding should be the same for top/bottom and left/right, otherwise your component will look weird.

Why are you using so many different sizes?

Reply:
Maybe I'm treating css as if it was Photoshop, 
I think I'll learn bootstrap after we finish the final project, or I'll learn proper design principles